Tests/ApodiniTests/MetadataTests/BuildinMetadata/DescriptionMetadataTests.swift

```diff
@@ -0,0 +1,134 @@
+//
+// Created by Lorena Schlesinger on 10.01.21.
+//
+
+import XCTest
+@testable import Apodini
+
+final class DescriptionModifierTests: ApodiniTests {
+    struct TestContent: Content {
+        static var metadata: Metadata {
+            Description("Content Description!")
+        }
+    }
+
+    struct TestHandler: Handler {
+        func handle() -> TestContent {
+            TestContent()
+        }
+
+        var metadata: Metadata {
+            Description("The description inside the TestHandler")
```

Comment:
You are pretty flexible in that regard. You may have seen those `.metadata` modifiers (either via ResultBuilder or directly passing a single Metadata instance) which can be used to overwrite Metadata on Handlers.
Additionally as mentioned before, you might choose to add a regular HandlerModifier for your Metadata.

One drawback currently is, that the `HandlerMetadataNamespace` is not available inside `Component`s, thus you can't really pass Metadata which is only defined for Handlers via the `.metadata` Modifier directly. However, as a workaround you can define a custom `HandlerMetadataGroup` and pass that via the `.metadata` Modifier.

Not sure if we should add the `HandlerMetadataNamespace` to the `Component` protocol as well. I think this could introduce some complications in regards to type checking and Metadata with similar names.